pom.xml

```diff
@@ -228,6 +229,11 @@
                 <version>${apache.kafka.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
I think this library is deprecated, I see this as the latest release for Kafka Serdes
```
<dependency>
    <groupId>io.apicurio</groupId>
    <artifactId>apicurio-registry-avro-serde-kafka</artifactId>
    <version>3.0.2</version>
</dependency>
```

Reply:
Fixed